src/txn/txn.c

```diff
@@ -2871,7 +2875,7 @@ __wt_verbose_dump_txn_one(
         ", prepare_timestamp: %s"
         ", pinned_durable_timestamp: %s"
         ", read_timestamp: %s"
-        ", checkpoint LSN: [%" PRIu32 "][%" PRIu32 "]"
```

Comment:
I want to flag that there's a few places where the printed string has changed format. Here from `checkpoint LSN: [1][128]` to `checkpoint LSN: [1,128]`, and in other places from `[1/128]` to `[1,128]`. 
This seems harmless. I'm not aware of any scripts or log parsing rules that depend on the format of these strings

Reply:
Agreed it'd be nice to have a single consistent format. I gave square bracket a try but `__wt_meta_ckptlist_set` needs circular brackets for the checkpoint config string. I'll leave this as is.